src/gui/group/EditGroupWidget.h

```diff
@@ -43,7 +43,7 @@ class IEditGroupPage
     virtual QString name() = 0;
     virtual QIcon icon() = 0;
     virtual QWidget* createWidget() = 0;
-    virtual void set(QWidget* widget, Group* tempoaryGroup) = 0;
+    virtual void set(QWidget* widget, Group* tempoaryGroup, const Database* database) = 0;
```

Comment:
This should be a ```QSharedPointer<Database>```, we do not want to pass around bare database pointers even if they are const.

Reply:
When you are passing the database you can use: ```std::move(m_db)``` to prevent a copy